spec/cftouk/_A_CFTO_UK.test.js

```diff
@@ -0,0 +1,123 @@
+var cfto = require("../../common/cftoFunctions")
+
+describe('Testing - CFTO UK Site', function() {
+
+    beforeEach(function(done) {
+        browser.deleteCookie();
+        this.timeout(0);
+        browser.url('http://christmasfood.sit2.marksandspencer.com');
+    });
+
+    it('Add an item to the basket', function() {
+        this.timeout(0);
+        cfto.addToBag();
+        cfto.logIn();
+    });
+
+    it('Place an UK CFTO order with registered user with added item', function() {
+        this.timeout(0);
+
+        //Add item to the basket & land on the basket page by click on the bag button on the minibag pop-up
+        cfto.addToBag();
+
+        // Function to login the user
+        cfto.logIn();
+
+        browser.click('button[data-analyticsfunction="analyticsCheckoutButtonClicked"]'); // This will appear in case of basket merge scenario
+
+        cfto.chooseCollectionUK(); // To choose the collection date & slot
+
+        // Add a new card from payment page
+        browser.click('#add-card-btn-inline');
+        cfto.cardDetailsLightBox(); // To put in the card details
+        browser.click('button[data-element="AddCardAddressOverlay"]');
+        browser.waitForVisible('#address-details__add');
+        browser.click('#address-details__add');
+        browser.waitForVisible('.payment-button');
+        browser.setValue('.credit-debit-card-form #cvv', '609');
+        browser.click('.payment-button');
+
+        // Confirm the order by clicking on the place order button
+        browser.click('input[type="submit"]');
+
+        //Confirming the details on the order confirmation page
+        expect(browser.element('.order-confirm-msg__content .order-confirm-msg__keyword').getText()).to.contain('401');
+    });
+
+
+    it('Place an UK CFTO order with guest user', function() {
```

Comment:
No, Sir guest user flow of IE is bit different than that of UK. In IE user cannot search a store. He directly lands to the store selection page. Hence, different functions is mentioned for UK & IE